components/src/components/cardList/index.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import Card from '../card/card';
+import style from './style.module.scss';
+import data from '../../data/data';
```

Comment:
create in folder data index.ts file 
And use just import data from '../../data';